doc/source/dev/roadmap-detailed.rst

```diff
@@ -334,26 +334,40 @@ is that they act like ``numpy.matrix`` (which will be deprecated in NumPy at
 some point).
 
 What we want is sparse arrays, that act like ``numpy.ndarray``. In SciPy
-``1.8.0`` a new set of classes (``csr_array`` et al.) has been added - these
-need testing in the real world, as well as a few extra features like 1-D array
-support.
-An alternative (more ambitious, and unclear if it will materialize at this
-point) plan is being worked on in https://github.com/pydata/sparse.  The
-tentative plan for that was/is:
-
-- Start depending on ``pydata/sparse`` once it's feature-complete enough (it
-  still needs a CSC/CSR equivalent) and okay performance-wise.
-- Add support for ``pydata/sparse`` to ``scipy.sparse.linalg`` (and perhaps to
-  ``scipy.sparse.csgraph`` after that).
-- Indicate in the documentation that for new code users should prefer
-  ``pydata/sparse`` over sparse matrices.
-- When NumPy deprecates ``numpy.matrix``, vendor that or maintain it as a
-  stand-alone package.
+``1.8.0`` a new set of classes (``csr_array`` et al.) was added.
+Construction functions for arrays were added in ``1.12.0``. Support for
+1-D array is expected in ``1.13.0``.
+
+Next steps toward sparse array support:
+
+- Extend sparse array api to 1-D arrays.
+    - support for coo, csr and dok formats.
+    - csr 1d support (using sparsetools) for min-max, indexing, arithmetic
+- Help other libraries convert to sparse arrays from sparse matrices.
+  Create transition guide and helpful scripts to flag code that needs
+  further examination. NetworkX, Scikit-learn and scikit-image are in
```

Comment:
typo: scikit-learn is lower-case. Also "numpy" -> "NumPy" if it's the project name.